TheOracle2/Interactions/MessageCommands/RecreateMessageCommand.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace TheOracle2.Commands;
 
-public class RightClickCommands : InteractionModuleBase<SocketInteractionContext>
+public class RecreateMessageCommand : InteractionModuleBase<SocketInteractionContext>
```

Comment:
more descriptive name, since it's a module of one command atm; add'l MessageCommands can live as their own modules in the MessageCommands directory